admin-http-gateway/src/main.rs

```diff
@@ -124,9 +124,10 @@ async fn main() -> Result<(), rocket::Error> {
     mc_common::setup_panic_handler();
     let _sentry_guard = mc_common::sentry::init();
 
+    let (logger, _global_logger_guard) = create_app_logger(o!());
```

Comment:
:x: I think this has to come before the panic handler, https://github.com/mobilecoinfoundation/mobilecoin/blob/master/common/src/panic_handler.rs#L33
That looks like the issue that happened in the stack trace.  A panic occurred and the panic handler tried to log.

Reply:
I don't think it matters as long as we don't panic before we create the logger, but I suppose since the panic handler relies on the logger being set up it does make more sense to first create the logger an then install the panic handler. So I'll change to that.